mercadolibre/meli-card-drawer-ios
# (Changed) [PXN-5161] - Adapting PANView for receiving an issuer image in split cases

## What have changed?

We added a container in the PANView for the incoming issuer image from split cases. If the case is not split, we'll not show that container.

## How to test:

In the ExampleApp -> Helpers -> CardUIExamples, we implemented the optional var `panStyle` of the `CardUI` protocol, and tested the different scenarios.

- If the integrator set the URL for the issuerImage correctly.
- If the integrator set nil for the `issuerImageUrl`.
- If the integrator set a empty String.
- If the integrator send a bigger image than expected.


## Prints for layout changes:

- If the integrator set the URL for the `issuerImageUrl` correctly

![Simulator Screen Shot - iPhone 13 mini - 2022-10-17 at 14 31 52](https://user-images.githubusercontent.com/96147063/196244590-da38d0e7-e23c-43d9-bcbe-275037af0fa3.png)

![Captura de Pantalla 2022-10-17 a la(s) 14 06 03](https://user-images.githubusercontent.com/96147063/196244645-bc146ec3-7ec7-4c05-b1f5-c271d8b1ea55.png)

- If the integrator set nil for the `issuerImageUrl`

![Simulator Screen Shot - iPhone 13 mini - 2022-10-17 at 14 22 29](https://user-images.githubusercontent.com/96147063/196244729-a53b6dc8-b661-4306-892f-c3d5dd7c3bf6.png)

![Captura de Pantalla 2022-10-17 a la(s) 14 07 26](https://user-images.githubusercontent.com/96147063/196244806-290d2156-2abe-4350-aaec-a9de8be48dac.png)


- If the integrator set a empty String

![Simulator Screen Shot - iPhone 13 mini - 2022-10-17 at 14 22 29](https://user-images.githubusercontent.com/96147063/196244902-8db92268-6135-4f39-9704-0c29d2522ab8.png)

![Captura de Pantalla 2022-10-17 a la(s) 14 06 47](https://user-images.githubusercontent.com/96147063/196244934-02f4fe5f-941c-4f8a-85a8-ade424ff9109.png)


- If the integrator send a bigger image than expected (original dimensions of the testingImage: 2000x1553)

![Simulator Screen Shot - iPhone 13 mini - 2022-10-17 at 14 24 06](https://user-images.githubusercontent.com/96147063/196244990-963e2820-6031-4fe1-a5f6-0e17412aadfd.png)

![Captura de Pantalla 2022-10-17 a la(s) 14 04 48](https://user-images.githubusercontent.com/96147063/196245027-8dc2d60b-8224-4a31-812f-8686a143178c.png)

## Extras:
